newzealidar/process.py

```diff
@@ -136,6 +142,59 @@ def gen_instructions(
     return instructions
 
 
+def store_vector_files_to_s3(s3_manager, s3_objects, instructions: dict):
+    data_paths = instructions["instructions"]["data_paths"]
+    vector_dir = pathlib.Path(data_paths["local_cache"]) / "vector" / data_paths["subfolder"]
+    vector_files = utils.get_files([""], vector_dir)
```

Comment:
Is the `[""]` intentional to get all files regardless of file extension? Would be good to add a comment

Reply:
Yes, I'm using the `get_files` function written by Xander in the `utils.py` script. I believe this function can retrieve all files regardless of their extension by using `[""]` unless you have a better suggestion than `[""]`. I will add a comment for this.

Reply:
A comment will be fine